epregressions/builds/base.py

```diff
@@ -40,8 +40,30 @@ def __init__(self):
     def get_idfs_in_dir(idf_dir: Path) -> Set[Path]:
         idf_path = Path(idf_dir)
         all_idfs_absolute_path = list(idf_path.rglob('*.idf'))
+        all_idfs_absolute_path.extend(list(idf_path.rglob('*.imf')))
         all_idfs_relative_path = set([idf.relative_to(idf_path) for idf in all_idfs_absolute_path])
-        return all_idfs_relative_path
+        known_ignore_list = [
+            # these files are for running EnergyPlus _as an FMU_ and we aren't doing that
+            '_ExternalInterface-actuator.idf',
+            '_ExternalInterface-schedule.idf',
+            '_ExternalInterface-variable.idf',
+            # these files are macro resource files, imported by AbsorptionChiller_Macro.imf
+            'HVAC3ZoneGeometry.imf',
+            'HVAC3ZoneMat-Const.imf',
+            'HVAC3ZoneChillerSpec.imf',
+            'HVAC3Zone-IntGains-Def.imf',
+        ]
```

Comment:
But ignore macro resource files.  And also ignore the external interface files that are set up to run EnergyPlus _inside_ an FMU.  Note we are actually running the external interface files that are setup to have EnergyPlus as the primary driver calling out to the prebuilt fmu objects.